src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py

```diff
@@ -70,6 +70,8 @@ def _get_default_install_location(exe_name):
 register_extra_cli_argument('acs create', 'generate_ssh_keys', action='store_true', help='Generate SSH public and private key files if missing')
 register_cli_argument('acs create', 'agent_vm_size', completer=get_vm_size_completion_list)
 
+register_cli_argument('acs create', 'windows', action='store_false', help='If true, deploy a windows container cluster.')
```

Comment:
why not `store_true`? This automatically creates a default value of False, which I believe is you want
`store_false` will default to True when the command-line argument is not present. Is that what you want?